src/wal.c

```diff
@@ -247,6 +247,8 @@
 ** that correspond to frames greater than the new K value are removed
 ** from the hash table at this point.
 */
+#include "pager.h"
```

Comment:
Is this creating a cyclic dependency between pager and wal? Was it before a clean "higher level depends on the lower" kind of dependency?

Reply:
oh I think this is just redundant leftover, I don't need to know about pager here, thanks

Reply:
it was redundant indeed - dropped